### PR TITLE
success page created

### PR DESCRIPTION
**Pull request by deviator53**
- created success page design

### CODE REVIEW

**Review by CreedTech — APPROVED** [review]

**Comment by CreedTech** [comment]
Approved

File: src/pages/SuccessPage.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Stack, Typography } from '@mui/material';
+import { Link } from 'react-router-dom';
+import { makeStyles } from "@mui/styles";
+
+
+const SuccessPage = () => {
```
Comment (Vinyl-Davyl) [inline src/pages/SuccessPage.tsx:7]: Use a "const SuccessPage:React.FC = () => {" here

**Review by Vinyl-Davyl — CHANGES_REQUESTED** [review]
Use a Functional Component in the Success page 
I left a comment there

File: src/pages/SuccessPage.tsx (hunk 1 above)
Comment (CreedTech) [inline src/pages/SuccessPage.tsx:7]: Functional component

**Review by CreedTech — COMMENTED** [review]
Functional component

**Pull request merged**